src/QMCWaveFunctions/Fermion/DelayedUpdate.h

```diff
@@ -39,15 +47,17 @@ class DelayedUpdate
   std::vector<int> delay_list;
   /// current number of delays, increase one for each acceptance, reset to 0 after updating Ainv
   int delay_count;
+  /// matrix inversion engine
+  DiracMatrix<T_FP, T> detEng;
 
 public:
   /// default constructor
   DelayedUpdate() : delay_count(0) {}
 
   /** resize the internal storage
-       * @param norb number of electrons/orbitals
-       * @param delay, maximum delay 0<delay<=norb
-       */
+   * @param norb number of electrons/orbitals
```

Comment:
@PDoakORNL I changed the alignment by hand. Is there a way to teach clang-format to do better?